src/components/LoanCards/LoanCardImage.vue

```diff
@@ -63,13 +63,40 @@ export default {
 		openInNewTab: {
 			type: Boolean,
 			default: false
+		},
+		imageEnhancementExperimentVersion: {
+			type: String,
+			default: ''
+		},
+		loanImageHash: {
+			type: String,
+			default: ''
 		}
 	},
+
 	computed: {
+		getImageRetinaUrl() {
+			return this.getImageUrl(this.retinaImageUrl, 'w_970,h_720,c_fill,g_faces');
+		},
+
+		getImageStandardUrl() {
+			return this.getImageUrl(this.standardImageUrl, 'w_480,h_360,c_fill,g_faces');
```

Comment:
Would it possibly make more sense to parse the width/height out of the retinaImageUrl and standardImageUrl props instead of forcing every image to be this size? Or is this done to limit the number of transformations that are performed?

Reply:
Sure, the parsing could be done. And it's not done to limit the number of transactions since once the transformation is complete, it's cached on Cloudinary's side.

Reply:
I was referring to that by parsing out the size, you could end up with more sizes being requested than just these 2, which would result in more transformations being performed.